dspace/config/launcher.xml

```diff
@@ -242,6 +242,13 @@
 	    <argument>reindex</argument>
         </step>
     </command>
+    <command>
+        <name>solr-upgrade-pre6x-statistics</name>
```

Comment:
Again, a minor thing, but I think we should use the same naming convention as other commands around Solr (e.g. `solr-reindex-statistics` and `solr-import-statistics` and `solr-export-statistics`).  So, I'd like to see the "pre6x" part put at the *end*... e.g. `solr-upgrade-statistics-pre6x`. That way we can have future `solr-upgrade-statistics-*` commands that get grouped together.

To be honest, I'm not even sure we need it to say `pre6x`.  It could just be `solr-upgrade-statistics-6x` , and then we change the <description> to say "Upgrade statistics (integer to UUID) to DSpace 6.x (or above) compatibility".